adapter/stackdriver/log/BUILD

```diff
@@ -25,5 +25,9 @@ go_test(
         "//adapter/stackdriver/config:go_default_library",
         "//pkg/adapter/test:go_default_library",
         "//template/logentry:go_default_library",
+        "@com_google_cloud_go//logging:go_default_library",
+        "@org_golang_google_api//option:go_default_library",
+        "@org_golang_google_genproto//googleapis/api/monitoredres:go_default_library",
```

Comment:
new dependencies without new code ?
Is this a minimal set?

Reply:
These are probably indirectly imported. It's better to be explicit.